pnparser/patch_notes.py

```diff
@@ -0,0 +1,625 @@
+from .exceptions import ParserError, ParserHttpError, ParserTimeoutError
+from .helpers import Helper, Filters
+from .dragon import Dragon
+from .templates import *
+
+from dateutil import parser as DatetimeParser
+from datetime import datetime as DateTime
+
+from bs4 import BeautifulSoup, Tag
+import requests as HttpClient
+
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
```

Comment:
I guess you looked at Tsubaki, those checks ("MonsterModel" etc) are only required because some of the imports are specifically cross-cog imports. Cross-cog imports cannot be done in Red and so we have to demote them to be done only in the IDE, i.e. when TYPE_CHECKING is active. In this case it's not required because these are both just libraries, so you don't need the if TYPE_CHECKING check here.

Reply:
I honestly thought of doing this in a way to minimize the number of unnecessary imports because I thought there would be some benefit to it, but yeah, I can just import those.

Reply:
I kept it only where I was getting an import cicle error.